examples/Makefile.am

```diff
@@ -6,10 +6,13 @@ AM_CPPFLAGS += -I$(top_builddir)/inc
 AM_CPPFLAGS += $(BOOST_CPPFLAGS)
 AM_CPPFLAGS += $(GSL_CFLAGS)
 AM_CPPFLAGS += $(GRVY_CFLAGS)
-AM_CPPFLAGS += $(ANN_CFLAGS)
 
 LIBS += $(GSL_LIBS)
 
+if HAVE_ANN
+AM_CPPFLAGS += $(ANN_CFLAGS)
+endif
+
```

Comment:
No reason other than it's with the other conditional `AM_CPPFLAGS` adjustments.  Want me to move it back up?